### PR TITLE
add a few Federalist screenshots and edit settings text

### PR DESCRIPTION
**Pull request by wslack**

### CODE REVIEW

File: pages/using-federalist.md [hunk 1]
```diff
@@ -20,15 +22,18 @@ The [18F method cards](https://methods.18f.gov/) are an overview of several diff
...
 ## Managing site settings
+![Federalist Settings Screenshot]({{site.baseurl}}/assets/federalist-settings.png)
 
 The settings option for a site lets you adjust how Federalist handles the site. These settings are for advanced functionality and most people will not need to change them.
 
-- **Default branch** This is the branch of the site's repository that is built and hosted as the production version of the site. Other branches are considered drafts. Changing this value will rebuild the site using a different branch.
-
-- **Static Site Engine** This controls if Federalist builds your site using Jekyll or Hugo, or simply hosts content you've already generated. If using a template, this should always be "Jekyll." The site engine should not be altered once your site is live. Contact the Federalist team for additional support.
+#### Basic Settings
+- **List Site Branch** This is the branch of the site's repository that is built and hosted as the production version of the site. Other branches are considered drafts. Changing this value will rebuild the site using a different branch.
```
Comment (jseppi) [inline pages/using-federalist.md]: `List` -> `Live`

File: pages/using-federalist.md [hunk 2]
```diff
@@ -20,15 +22,18 @@ The [18F method cards](https://methods.18f.gov/) are an overview of several diff
...
 
-- **Custom domains** Enter domain (including protocol; without a trailing slash) to host your website on a custom domain. For example, `https://example.gov`. When this setting is defined, Federalist will change the "View" link for your site to use the custom domain. It will also remove the `baseurl` when generating your site, so that relative links are relative to the root domain. For more about setting a custom domain, see the "[launch checklist]({{site.baseurl}}/pages/using-federalist/launch-checklist)" section.
+- **Custom domains** Enter domain (including https://; without a trailing slash) to host your website on a custom domain. For example, `https://example.gov`. When this setting is defined during the launch process, Federalist will build your site so that links and images are loaded at the custom domain, so that relative links are set according to the custom domain. For more about setting a custom domain, see the "[launch checklist]({{site.baseurl}}/pages/using-federalist/launch-checklist)" section.
```
Comment (jseppi) [inline pages/using-federalist.md]: `...domain, so that...` -> `...domain, and so that...`

File: pages/using-federalist.md [hunk 3]
```diff
@@ -20,15 +22,18 @@ The [18F method cards](https://methods.18f.gov/) are an overview of several diff
...
 
 - **Demo Site** Federalist allows you to specify a different branch and domain for a "demo site" that can be used to publicly preview your content at a URL of your choice, such as "preview.login.gov." Launch of a demo URL works just like a live URL from a security standpoint.
 
+#### Advanced Settings
+- **Static Site Engine** This controls if Federalist builds your site using Jekyll or Hugo, or simply hosts content you've already generated. If using a template, this should always be "Jekyll." The site engine should not be altered once your site is live. Contact the Federalist team for additional support.
```
Comment (jseppi) [inline pages/using-federalist.md]: Maybe `If using one of the Federalist provided templates, this ...`

**Comment by wslack** [comment]
@jseppi fixed the typos

**Pull request merged**